### PR TITLE
replace default config string with const value

### PR DESCRIPTION
**Pull request by Mulavar**
**What this PR does**:
replace default config string with const value

**Which issue(s) this PR fixes**:

Fixes #
none

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by AlexStocks — APPROVED** [review]

File: common/constant/default.go [hunk 1]
```diff
@@ -94,3 +94,14 @@ const (
...
+
+// default config value
+const (
+	DEFAULT_REGISTRY_ZK_ID       = "demoZK"
+	DEFAULT_REGISTRY_ZK_PROTOCOL = "zookeeper"
+	DEFAULT_REGISTRY_ZK_TIMEOUT  = "3s"
+	DEFAULT_REGISTRY_ZK_ADDRESS  = "127.0.0.1:2181"
+
+	DEFAULT_PROTOCOL_DUBBO_NAME = "dubbo"
+	DEFAULT_PROTOCOL_DUBBO_PORT = "20000"
```
Comment (zouyx) [inline common/constant/default.go:106]: default port：https://github.com/apache/dubbo-go/blob/260fde4ad23d6cd9cb421cc6b9753ca01249e73f/common/constant/default.go#L47

File: common/constant/default.go [hunk 2]
```diff
@@ -94,3 +94,14 @@ const (
...
 )
+
+// default config value
+const (
+	DEFAULT_REGISTRY_ZK_ID       = "demoZK"
+	DEFAULT_REGISTRY_ZK_PROTOCOL = "zookeeper"
+	DEFAULT_REGISTRY_ZK_TIMEOUT  = "3s"
+	DEFAULT_REGISTRY_ZK_ADDRESS  = "127.0.0.1:2181"
+
+	DEFAULT_PROTOCOL_DUBBO_NAME = "dubbo"
```
Comment (zouyx) [inline common/constant/default.go:105]: default protocol：https://github.com/apache/dubbo-go/blob/260fde4ad23d6cd9cb421cc6b9753ca01249e73f/common/constant/default.go#L38

File: common/constant/default.go [hunk 3]
```diff
@@ -94,3 +94,14 @@ const (
 	DEFAULT_LOG_CONF_FILE_PATH      = "../profiles/dev/log.yml"
 	DEFAULT_ROUTER_CONF_FILE_PATH   = "../profiles/dev/router.yml"
 )
+
+// default config value
+const (
+	DEFAULT_REGISTRY_ZK_ID       = "demoZK"
+	DEFAULT_REGISTRY_ZK_PROTOCOL = "zookeeper"
```
Comment (zouyx) [inline common/constant/default.go:101]: use exist key like
```
DEFAULT_REGISTRY_ZK_PROTOCOL = ZOOKEEPER_KEY
```
reference：https://github.com/apache/dubbo-go/blob/260fde4ad23d6cd9cb421cc6b9753ca01249e73f/common/constant/key.go#L188

**Review by zouyx — COMMENTED** [review]
Can you use exist key in this PR?

**Review by zouyx — APPROVED** [review]

**Review by wongoo — APPROVED** [review]

**Review by zhaoyunxing92 — APPROVED** [review]
approved these changes

**Pull request merged**